### PR TITLE
Blog post by me: convos-solves-irc-persistence-problem.md

### PR DESCRIPTION
**Pull request by ivan-avalos**
in this pull request, I add a blog post written by me.

### CODE REVIEW

File: blog/2021/2021-01-07-convos-solves-irc-persistence-problem.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+title: Convos solves IRC's persistence problem
+author: Iván Ávalos
```
Comment (jhthorsen) [inline blog/2021/2021-01-07-convos-solves-irc-persistence-problem.md:3]: As we talked about on IRC: If you want to post this on your own web site, then we can add a canonical here to inform the search engines. Like this:

```
title: ...
author: ...
canonical: https//yoursite/blog/whatever
```

Here is an example where we also added a link in the introduction: https://raw.githubusercontent.com/Nordaaker/convos/www.convos.chat/blog/2020/2020-11-21-toroid-org-convos-rocks.md

**Comment by jhthorsen** [comment]
Thanks 👍

**Comment by serhack** [comment]
By the way, I do not want to come with drama but the post looks similar to mine. https://serhack.me/articles/convos-web-client-persistent-irc/

**Comment by ivan-avalos** [comment]
@serhack True! Lol. I swear I had never read your blog post!

**Comment by marcusramberg** [comment]
Honestly I don't see any problem here. There are bound to be some similarities in these kinds of posts. I appreciate both of you taking the time to contribute. <3

**Pull request merged**